arch/arm64/boot/dts/qcom/msm8953-xiaomi-common.dtsi

```diff
@@ -170,6 +170,25 @@
 
 		status = "disabled";
 	};
+
+	gt917d_ts: touchscreen@5d {
+		compatible = "goodix,gt917d";
+		reg = <0x5d>;
+
+		interrupt-parent = <&tlmm>;
+		interrupts = <65 IRQ_TYPE_EDGE_FALLING>;
```

Comment:
I think it should be ```IRQ_TYPE_LEVEL_LOW``` downstream uses 0x2008 for focaltech it uses 0x2002

Reply:
I don't remember anymore why we chose edge falling. We thought about 0x2008 for a while. @apolemus just told me that it is `IRQF_ONESHOT | IRQ_TYPE_LEVEL_LOW`. So should I change it to `IRQF_ONESHOT | IRQF_PROBE_SHARED | IRQ_TYPE_LEVEL_LOW`?

Reply:
We chose IRQ_TYPE_EDGE_FALLING because downstream _interrupts_ is [0x2](https://github.com/RebornDaisyProject/daisy_msm8953/blob/42bff3717bb0a38774291a999795dfcd28bb2434/arch/arm64/boot/dts/qcom/daisy/daisy-qrd.dtsi#L167) where we concluded that it should be falling.